ui/hooks/experiences/useRamps/useRamps.constants.ts

```diff
@@ -0,0 +1,89 @@
+import { CHAIN_IDS } from '../../../../shared/constants/network';
+
+export const portfolioUrl = process.env.PORTFOLIO_URL;
+export const buyPath = '/buy';
+export const entryParam = 'metamaskEntry';
+export const entryParamValue = 'ext_buy_button';
+
+export const MANUALLY_ACTIVE_CHAIN_IDS = [CHAIN_IDS.SEPOLIA];
+
+export const mockedNetworks = [
```

Comment:
Suggestion: we should move the mock data close to tests, or in the test, while `constants` will be values used within feature code base

Reply:
Agreed that we should keep a clear separation between test and production code, if it's not too inconvenient at least. The less code we ship to users the better.

I don't think this exact topic was discussed or any consensus reached on it; we had discussed a related subject (test ID handling) but that's not quite the same because those _are_ used in both test and production code.

Reply:
OK. I moved the mocked data to the test file in [3efe742](https://github.com/MetaMask/metamask-extension/pull/18862/commits/3efe74283300fbc56b85f4893e2241fffa724d59)